SCAutolib/models/file.py

```diff
@@ -298,6 +297,14 @@ def __init__(self):
             return
         self.__initialized = True
 
+        with open('/etc/redhat-release', "r") as f:
+            release = f.read()
+
+        if "release 8" in release or "release 9" in release:
+            self._template = Path(TEMPLATES_DIR, "sssd.conf-old")
```

Comment:
If "sssd.conf-old" file contains a template for earlier versions of SSSD, then it would be better to name it accordingly, like "sssd.conf-rhel8"
```suggestion
            self._template = TEMPLATES_DIR / "sssd.conf-old"
```